pandas-stubs/core/series.pyi

```diff
@@ -183,6 +184,12 @@ from pandas.core.dtypes.dtypes import CategoricalDtype
 
 from pandas.plotting import PlotAccessor
 
+_T_INT = TypeVar("_T_INT", bound=int)
+_T_COMPLEX = TypeVar("_T_COMPLEX", bound=complex)
+_T_INT_FLOAT = TypeVar("_T_INT_FLOAT", bound=int | float)
+_T_FLOAT_COMPLEX = TypeVar("_T_FLOAT_COMPLEX", bound=float | complex)
+_T_INT_FLOAT_COMPLEX = TypeVar("_T_INT_FLOAT_COMPLEX", bound=int | float | complex)
```

Comment:
aren't `_T_FLOAT_COMPLEX`, `_T_COMPLEX`, and `_T_INT_FLOAT_COMPLEX` from a typing persepctive the same?

Reply:
A function that accepts a complex, is also assumed (from a typing perspective) to accept a float or an int https://peps.python.org/pep-3141/#abstract even though they aren't sub-classes at runtime